src/style.css

```diff
@@ -1 +1,82 @@
-/* Styles go here */
+* {
+  vertical-align: baseline;
+  font-weight: inherit;
+  font-family: inherit;
+  font-style: inherit;
+  font-size: 100%;
+  border: 0 none;
+  outline: 0;
+  padding: 0;
+  margin: 0;
+}
+
+.container {
+  width: 1200px;
+  margin: 0 auto;
+}
+
+.header {
+  background-color: #fff;
+ 
+}
+
+.header__inner {
+  /* border: 1px solid #000; */
+  padding: 10px 50px;
+  text-transform: uppercase;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  text-align: center;
+}
+
+.header__list {
+  display: flex;
+  align-items: center;
+  list-style: none;
+}
+
+.header__item {
+  text-decoration: none;
+  align-content: center;
+}
+
+.header__item:not(:last-child) {
+  margin-right: 20px;
+}
+
+.header__link {
+  font-family: "Roboto", sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 14px;
+  color: #000;
+  text-decoration: none;
+  display: flex;
+  transition: all 0.3s;
+}
+
+.header__link:hover {
+  color: #00acdc;
```

Comment:
move repeatable colors in the variable

Reply:
FIx it 
![image](https://user-images.githubusercontent.com/97547824/210101761-7aeb00fb-ce24-4455-83d6-efd6e5be8a2a.png)